package.json

```diff
@@ -16,7 +16,8 @@
         "infusion": "3.0.0-dev.20180222T160835Z.6e1311a",
         "node-jqunit": "1.1.8",
         "request": "2.88.0",
-        "winstrap": "0.5.12"
+        "winstrap": "0.5.12",
+        "ws": "6.1.2"
```

Comment:
I'm curious, where is this dependency coming from?

Reply:
Oh, nevermind, I'm just seeing that we're using it in the gpiiConnector.

Reply:
@amb26 I see that kettle is using 6.0.0 while we're adding 6.1.2 in this pull request. I don't see any problem with that, but just wanted to double check with you in case you see any potential issue or if we would like to use the same version in both sides.

Reply:
Hi @javihernandez - this is fine - ws artefacts never get shared across the module boundary. In any case Kettle is tested fine with 6.1.x in this pull in progress - https://github.com/fluid-project/kettle/pull/49

Reply:
okay, thanks for confirming